src/Sarif/OptionallyEmittedData.cs

```diff
@@ -70,6 +70,6 @@ public enum OptionallyEmittedData : uint
         // information in the SARIF log file. In the absence of this setting, any existing
         // data that would otherwise have been overwritten by the insert operation will
         // be preserved.
-        OverwriteExistingData = 0x80000000
+        OverwriteExistingData = 0x8000000
```

Comment:
>0x8000000 [](start = 32, length = 9)

Same thing, please use 0x40000000 for this value...

Reply:
feel free to merge after taking feedback

---
In reply to: [277882305](https://github.com/Microsoft/sarif-sdk/pull/1454#discussion_r277882305) [](ancestors = 277882305)